### ISSUE TITLE
Button to reset profile picture back to the default one

### ISSUE THREAD

**Issue by ghost** [qTox/qTox#3246]
I think that there should be a button on one's profile page to reset their profile picture back to the default qTox one it starts of with for the following reasons:
- If one changes their profile picture to something personal or just something they need to change it away from quickly they will be able to quickly press a button to change it back and minimize the impact of such an unfortunate error
- If one wants to set their profile picture back to the default one they won't have to start looking through all the qTox installation folders to find it and will be able to just set it easily

**Comment by zetok** [comment]
Note:
- implementation needs to not clutter profile page
- when removing current profile image user needs to be asked for confirmation

**Comment by Diadlo** [comment]
What problem with 
Right mouse button click on photo -> Remove ?

**Comment by zetok** [comment]
Right. The obvious problem with the feature is that it's hidden in plain sight in a way that made everyone forget that it even exists.

So I think that adding a tooltip that would say that one can remove picture via r-click would do the job.

**Comment by Diadlo** [comment]
@zetok what about add button on mouse hover?
![spectacle f11990](https://cloud.githubusercontent.com/assets/6306608/15043273/9efefe14-12dc-11e6-9696-475ac2d3e7b0.png)

**Comment by Diadlo** [comment]
@zetok what you think about add button on mouse hover?
![spectacle f11990](https://cloud.githubusercontent.com/assets/6306608/15043340/3c0f5b54-12dd-11e6-922f-970cc5cc0579.png)

**Comment by ghost** [comment]
What you have now is probably better as it is less and more subtle, but just a suggestion, upon clicking on the profile picture instead of bringing up the file chooser perhaps it could bring up another window which allows you to either click the "Remove" button or the "New picture" (or something like that) button which allows you to set it with the file chooser. These buttons would be next to a larger version of the current profile picture which perhaps it should allow you to resize (like can be done on Google and FaceBook etc - just to not make having an extra window pointless and to actually make it add more functionality than a remove feature).

Of course this is probably more something for a new issue, but just thought I'd throw it in there.

**Comment by ProMcTagonist** [comment]
@SeventhSonOfASeventhSon too many clicks

@Diadlo I like that

**Comment by zetok** [comment]
@Diadlo Button is a bit ~small – for some people it's hard to click on small things.

What I was wondering about is something like that on mouse hover:
![20160506 qtox profile removal](https://cloud.githubusercontent.com/assets/3148759/15065943/a4ca7a96-135c-11e6-89da-1fe48d384932.png)
(with "background" colours even more transparent, and perhaps `× / +` signs with some sort of highlight or frame)

**Comment by ProMcTagonist** [comment]
@zetok I don't think removing the profile picture warrants the same amount of click and eye space as changing the profile picture.

**Comment by SkyzohKey** [comment]
What I'm gonna implement for Ricin, you could probably do something similar:
![Delete avatar button on avatar hover](https://a.uguu.se/kfxnbe.svg)

**Comment by zetok** [comment]
@ProMcTagonist hm, right. What about something like a mix of @SkyzohKey's idea and mine? I.e. some 25% of space for ~transparent `Delete` button, and other 75% of space for ~transparent `Add` button?

**Comment by ghost** [comment]
@zetok, Yes, I think some thing like what you say would be good, however I think that covering the entire picture like you had in your see-through buttons version above makes it look rather messy, all over the place and ugly. So I think that something more subtle is needed, but I like @SkyzohKey's idea, may just the add button should be more subtle, or somehow not have it all on the profile picture, I don't know.

**Comment by ghost** [comment]
I think a more balanced but not completely imposing option is needed.
